modules/local/create_bed.nf

```diff
@@ -25,7 +25,7 @@ process CREATE_BED {
 
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":
-        create_bed: 1.01
+        create_bed: 1.02
```

Comment:
In line 24, will `sed` be able to handle arguments? Maybe remove `args` unless you are expecting something specific.